Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM node:14.17.6
-RUN mkdir -p /app
-WORKDIR /app
-COPY package*.json /app
-RUN npm install
-COPY .  /app
-CMD [ "npm",  "run", "start"]
\ No newline at end of file
+FROM node:14.17.6
+RUN mkdir -p /app
+WORKDIR /app
+COPY package*.json /app/
+RUN npm install
+COPY .  /app/
+CMD [ "npm",  "run", "serve"]
```

Comment:
Btw, you will need to have separate docker file for development and production (not now, just mark for the future). Because current one is for development, production should use build + start.